docs/api-reference.md

```diff
@@ -81,51 +261,53 @@ This function works similarly to `FOCUS()`, except instead of marking a block as
 !!!note
 	`SKIP` does not work inside an `it` block. The bodies of these blocks aren't executed until the tests run, which is too late to change which tests will run.
 
-### expect
+### itFOCUS, itSKIP, and itFIXME
 ```
-expect(value)
+itFOCUS(phrase: string, callback(context: table))
+itSKIP(phrase: string, callback(context: table))
+itFIXME(phrase: string, callback(context: table))
 ```
 
-Creates a new `Expectation`, used for testing the properties of the given value.
-
-Expectations are intended to be read like English assertions. These are all true:
-
-```lua
--- Equality
-expect(1).to.equal(1)
-expect(1).never.to.equal(2)
-
--- Approximate equality
-expect(5).to.be.near(5 + 1e-8)
-expect(5).to.be.near(5 - 1e-8)
-expect(math.pi).never.to.be.near(3)
+These methods are special versions of `it` that automatically mark the `it` block as *focused* or *skipped*. They're necessary because `FOCUS`, `SKIP`, and `FIXME` can't be called inside `it` blocks!
```

Comment:
Mention `fit`, `xit`, `fdescribe` and `xdescribe` somewhere around here?

Reply:
forgot about these!